pages/print_labels/control_plates.vue

```diff
@@ -16,7 +16,6 @@
             id="numberOfBarcodes"
             v-model="numberOfBarcodes"
             type="number"
-            value="1"
             min="1"
```

Comment:
I can't visualise multiple barcodes fully but looks OK.

Reply:
The `v-model` and the `value` attributes were conflicting each other.  In Vue 2 this didn't cause a problem, but in Vue 3, there must be an update order issue.  You could change it to 2 in the UI and the model would store 2, then when you tried to change it to 3, it would flick back to 1.  The value isn't needed because the model object stores the value and is initialised to 1 anyway.